src/main/java/com/three_tech_solutions/slot_app/data/mappers/StudentMapper.java

```diff
@@ -62,6 +62,7 @@ public StudentDetailsResponse toStudentDetailsResponse(Student student) {
                 student.getAdmissionDate(),
                 student.getPaymentPlan().getPaymentPlanName().getName(),
                 student.getPaymentPlan().getPlan().getName(),
+                (int) student.getPaymentPlan().getPlan().getNumberOfDays(),
```

Comment:
Y esto acá ya lo podríamos quitar


```suggestion
                student.getPaymentPlan().getPlan().getNumberOfDays(),
```